students/synowiec_krzysztof/lesson_01_basics/clockFaceOne.py

```diff
@@ -0,0 +1,11 @@
+def main():
+    anglePerSecond = (3600 * 12)/360
```

Comment:
it's okay, but sometimes it's good to write like:
`anglePerSecond` = (60 * 60 * 12)/360`
to make thing easier to understand and show how some values are build